README.md

```diff
@@ -210,7 +212,7 @@ Once the environment variable "TAIPY_GUI_DIR" is set (as an environment variable
   This command will fail if the environment variable "TAIPY_GUI_DIR" is not set properly.
 
 The 'front-end' directory will have an additional subdirectory called 'node_modules' where
-all dependent libraries are installed.
+all dependent libraries are installed.<br>
```

Comment:
This <br> is useless.
(and by the way we use XHTML syntax everywhere for easier potential parsing)